networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/ActiveEth2P2PNetwork.java

```diff
@@ -179,11 +179,17 @@ public void onSyncStateChanged(final boolean isInSync, final boolean isOptimisti
     gossipForkManager.onOptimisticHeadChanged(isOptimistic);
   }
 
-  private boolean isCloseToInSync() {
+  @VisibleForTesting
+  boolean isCloseToInSync() {
+    final UInt64 currentEpoch = recentChainData.getCurrentEpoch().orElseThrow();
```

Comment:
I am not sure if it is safe to use `orElseThrow()` here.

The method `isCloseToInSync()` is used as part of the startup process of `ActiveEthP2PNetwork` and depends on RecentChainData.
Although I believe we wait for successful initialization of RecentChainData before starting other services like p2p, we would be safer handling the unavailability of `recentChainData.getCurrentEpoch()` by returning false.